src/ApiCore/Transport/GrpcTransport.php

```diff
@@ -68,6 +88,8 @@ class GrpcTransport extends BaseStub implements TransportInterface
      *
      *    @type array $stubOpts Options used to construct the gRPC stub.
      *    @type Channel $channel Grpc channel to be used.
+     *    @type UnaryInterceptorInterface[] $interceptors *INTERNAL* Interceptor support, required until
```

Comment:
Should https://github.com/googleapis/gax-php/pull/182/files#diff-f84fac5ef33d7480e5a575b4b95f04dfR72 match this description as well? Also, WDYT about using the term "experimental" as we do elsewhere to indicate the type could potentially change?

Reply:
Done. Agree that experimental is better.